openstack/utils/templates/snippets/_set_ingress_cors_annotations.tpl

```diff
@@ -0,0 +1,10 @@
+{{ define "utils.snippets.set_ingress_cors_annotations" }}
+{{- if .Values.cors.enabled }}
+ingress.kubernetes.io/enable-cors: "true"
+ingress.kubernetes.io/cors-allow-headers: "{{ .Values.utils.cors.allow_headers }}{{ if .Values.cors.additional_allow_headers }},{{ .Values.cors.additional_allow_headers }}{{ end }}"
+ingress.kubernetes.io/cors-expose-headers: "{{ .Values.cors.expose_headers | default "*" }}"
+ingress.kubernetes.io/cors-allow-origin: "{{ .Values.utils.cors.allow_origin }}{{if .Values.cors.additional_allow_origin }},{{ .Values.cors.additional_allow_origin }}{{ end }}"
```

Comment:
Since we set now the `allow_origin` variable, I would remove `additional_allow_origin`. While it doesn't hurt to have additional origins, I am a fan of having only one way to achieve the desired result to keep the variance smaller.

Reply:
The point with `additional_allow_origin` was that a service might want more than the default. To achieve this, the service would have to duplicate the default into its own values files, which makes it harder to update the elektra development origins. Therefore, I wanted to provide a way to specify more for the parent chart like we do with `additional_allow_headers`.